Rome-Blockchain-Labs/terminal-widgets
# Fix wallet handling

- [x] RT-995 Add scrolling to wallet connect desktop list 
- [x] RT-996 Add error handling for failed wallet connections
- [x] RT-998 User has to press twice to connect to wallet
- [x] RT-999 Uniswapv2 widget goes blank after connection
- [x] RT-1001 Put token warning after wallet connection

To test

1. Select wallet connect as wallet and select desktop . expect scrollable list
2. Select wallet connect and then close. expect error message to be displayed
3. Open two widgets of different networks. Try toggling metamask between 2 networks. expect that user can switch between widgets without having to click twice on metamask
4. Open Uniswapv2 and another widget. Try toggling between the two networks. Uniswapv2 should never be a blank page
5. Expect that token warning only appears after connecting to a wallet
